packages/react-router-dom/docs/api/Prompt.md

```diff
@@ -1,3 +1,3 @@
 # &lt;Prompt>
 
-TODO
+Re-exported from core [`Prompt`](../../../react-router/docs/api/Prompt.md)
```

Comment:
@ryanflorence Can you confirm this is right? It's how all the other files were but it wasn't importing correctly for me. I'm assuming there's some build step I wasn't running.